Benchmark/hop_count/kilobot.cpp

```diff
@@ -0,0 +1,87 @@
+#pragma once
+#include "../../../Swarm-robotics-benchmark/Driver/Kilobot_simulation/kilobot_driver.h"
+
+START_USER_PROGRAM
+
+Channel *channel;
+Publisher *publisher;
+Subscriber *subscriber; 
+unsigned char msg1[10];

```

Comment:
msg1 is a name I randomly give when I was debugging, can you think of names that are more precise?